tools/apigee-x-trial-provision/README.md

```diff
@@ -127,3 +127,180 @@ the `CERTIFICATES` environment variable:
+        --mtu=1460
+    ```
+
+    Output:
+
+    ```sh
+    Created [https://www.googleapis.com/compute/v1/projects/qwiklabs-gcp-01-da1a8d6ae8b2/global/networks/exco-vpc].
+    NAME      SUBNET_MODE  BGP_ROUTING_MODE  IPV4_RANGE  GATEWAY_IPV4
+    exco-vpc  CUSTOM       REGIONAL
+    ...
+    ```
+
+1. Firewall Rules for internal, 22, and icmp traffic
+
+    ```sh
+    gcloud compute firewall-rules create $NETWORK-allow-internal --network $NETWORK --allow tcp,udp,icmp --source-ranges $NETWORK_CIDR
+    ```
+
+    Output:
+
+    ```sh
+    Creating firewall...⠹Created [https://www.googleapis.com/compute/v1/projects/qwiklabs-gcp-01-da1a8d6ae8b2/global/firewalls/exco-vpc-allow-internal].
+    Creating firewall...done.                                                                                                
+    NAME                     NETWORK   DIRECTION  PRIORITY  ALLOW         DENY  DISABLED
+    exco-vpc-allow-internal  exco-vpc  INGRESS    1000      tcp,udp,icmp        False
+    ```
+
+    ```sh
+    gcloud compute firewall-rules create fr-$NETWORK-ssh --network $NETWORK --allow tcp:22
+    ```
+
+    Output:
+
+    ```sh
+    Creating firewall...⠹Created [https://www.googleapis.com/compute/v1/projects/qwiklabs-gcp-01-da1a8d6ae8b2/global/firewalls/fr-exco-vpc-ssh].
+    Creating firewall...done.                                                                                                
+    NAME             NETWORK   DIRECTION  PRIORITY  ALLOW   DENY  DISABLED
+    fr-exco-vpc-ssh  exco-vpc  INGRESS    1000      tcp:22        False
+    ```
+
+1. GCP: Create a network subnet
+
+      ```sh
+      gcloud compute networks subnets create $SUBNET \
+          --network=$NETWORK \
+          --range=$SUBNET_CIDR \
+          --region=$REGION
+      ```
+
+      Output:
+
+      ```sh
+      Created [https://www.googleapis.com/compute/v1/projects/qwiklabs-gcp-01-da1a8d6ae8b2/regions/us-central1/subnetworks/exco-vpc-dev-subnet].
+      NAME                 REGION       NETWORK   RANGE
+      exco-vpc-dev-subnet  us-central1  exco-vpc  10.0.0.0/16
+      ```
+
+### Apigee X Org and GXLB
+
+1. Install required utilities
+
+    ```sh
+    sudo apt install -y git jq
+    ```
+
+1. Fetch apigee-x-provision utility
+
+    ```sh
+    git clone https://github.com/apigee/devrel.git
+
+    cd ~/devrel/tools/apigee-x-trial-provision/
+    ```
+
+1. With Google-managed certificate
+
+    ```sh
+    ./apigee-x-trial-provision.sh \
+    --network $NETWORK \
+    --subnet $SUBNET \
+    --region $REGION \
+    --zone $ZONE \
+    --ax-region $AX_REGION \
+    --certificates=managed
+
+    ```
+
+1. Output of the provisioning command will provide you with a test request you can use to verify functionality of the Apigee X organization.
+
+## Troubleshooting section
+
+### Send request to a private endpoint
+
+1. In the GCP Console, Open Compute Engine/Instance templates page.
+
+1. Click at the apigee-proxy-us-central1 template to open its properties.
```

Comment:
Click on the `apigee-proxy-[region]` template

Reply:
fixed